oddbird/oddleventy
# Fetch Mail archives on build

![](https://source.unsplash.com/featured/?cute,animal&build)

## Description
Fetches mail archives on build, so that it isn't blocked by user's ad block.

## Related Issue(s)
Closes #655 


## Steps to test/reproduce
Visit https://deploy-preview-661--oddleventy.netlify.app/oddnews/, and see that there are no network requests to https://oddbird.us19.list-manage.com
